Update README (#177)
brafdlog/budget-tracking: README.md

Internally it uses the [Israeli bank scrapers](https://github.com/eshaham/israeli-bank-scrapers) npm package.

### What is currently supported:

- One click to fetch transactions from multiple Israeli banks and credit cards
- Exporting transactions to Google sheets, YNAB, CSV or JSON file
- A basic UI for configuration
- A configuration for automatically classifying transactions to categories according to predefined patterns

## Running the app
You can download the app from [here](https://github.com/brafdlog/budget-tracking/actions?query=branch%3Amaster+workflow%3ABuild%2FRelease+is%3Asuccess+event%3Apush)

### Initial setup

> Before running the app for the first time you will need to set up the accounts you want to fetch data from (importers).

Reviewer: ```suggestion
The first time you run the app, you will need to set up the accounts you want to fetch data from (importers).
```
